e2e/routes/seeds/generated-credentials.test.js

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from "@playwright/test";
+import seedCredentials from "../../../public/temp/seed-credentials.json" assert { type: "json" };
```

Comment:
@wajeshubham  what you suggest we should skip that route for testing? 
CC: @jwala-anirudh

Reply:
No need to skip, if we can run the `e2e/routes/seeds/generated-credentials.test.js` test file at the end, then this would work. Because other seeding apis will populate this file. Let me know if that is possible.

Reply:
@wajeshubham okey, let me check with playwright how to change order of test files. update you once find solution

Reply:
@wajeshubham, 
I have checked playwright runs file in alphabetical order. 
it will run `ecommerce.test.js` first and then `generated-credentials.test.js`  we don't have to change anything.
I changed logic in **generated-credentials.test.js** file instead of importing json file it's read runtime 

CC: @jwala-anirudh